### PR TITLE
My stakeholder id percept

### PR DESCRIPTION
**Pull request by klikooo**
With this we can percept my_stakeholder_id(<ID>)

User story: 
As a developer, I want the virtual human to be able to see who it is, so that it can query information about itself, its indicators and its belongings.

### CODE REVIEW

File: environment/src/main/java/tygronenv/TygronEntity.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public TygronEntity(EisEnv env, String intendedStakeholder, Integer slotID) {
 	 * @throws EntityException
 	 */
 	public void notifyReady(String entity) throws EntityException {
-		connectStakeholder();
+	//	connectStakeholder();
```
Comment (Denpeer) [inline environment/src/main/java/tygronenv/TygronEntity.java]: old code?

Comment (klikooo) [inline environment/src/main/java/tygronenv/TygronEntity.java]: Yes will remove this,

**Comment by MaxGroenenboom** [comment]
Tested and it works, I approve.

**Comment by MarcianoJ** [comment]
I've watched the test and can verify it works, I approve

**Pull request merged**